tests/test_readme.py

```diff
@@ -64,6 +65,16 @@ def test_chat_with_model():
     assert "What food do llamas eat?" in result.stdout
 
 
+@RunIf(min_cuda_gpus=1)
+@mock.patch.dict(os.environ, {"LT_ACCELERATOR": "gpu"})
```

Comment:
As I understand, you did similar patching, but with `CPU`, so a test will run on a CPU even if it has access to a GPU.
But here it's not needed, since if you run a test only on a machine with a GPU, fabric should automatically select a GPU.

Reply:
Thanks! I must admit that I don't fully understand the testing procedures ... it's one of the most complicated things for me regarding litgpt.